### DISCUSSION TITLE
MQTT non si avvia

### DISCUSSION THREAD

**Discussion by 4sp3c** [indomus/forum#2246] [HUB: Home Assistant]
Ciao a tutti e scusate di nuovo. Dopo aver reinstallato HASSIO ho installato anche i componenti aggiuntivi tra cui MQTT. Quando faccio avvia, si avvia e mi fornisce anche il consumo della memoria e della CPU, ma non lo trovo poi su dispositivi e servizi. Ritornando su compnenti aggiuntivi trovo di nuovo che è in stop. In cosa sbaglio? dove posso vedere perché appena esco dalla schermata dei componenti mi torna in stop?
grazie a tutti

**Comment by snakuzzo** [comment]
Su dispositivi e servizi è dove abiliti l'integrazione con il broker MQTT, non c'entra niente con il fatto che parta o meno l'addon mosquitto (immagino tu stia usando quello).
Se l'addon parte e poi si ferma, devi controllare nei log dell'addon e/o del supervisor

**Reply by 4sp3c** [reply]
> Su dispositivi e servizi è dove abiliti l'integrazione con il broker MQTT, non c'entra niente con il fatto che parta o meno l'addon mosquitto (immagino tu stia usando quello). Se l'addon parte e poi si ferma, devi controllare nei log dell'addon e/o del supervisor

grazie mille davvero

**Comment by 4sp3c** [comment]
errata corrige... trovato dove è il log, trovato dove era l'errore... dentro accesscontrollist avevo scritto un topic # senza lasciare spazio tra i due

**Reply by snakuzzo** [reply]
ok...meglio così.
Ma non creare nuovi topic...fai reply sotto ultimo messaggio
